ITE/10/README.adoc

```diff
@@ -99,25 +99,26 @@ checkout step.
 
 === Setting Predicate Type Expectations for Steps and Inspections
 
-Currently, step declarations have the following schema.
+Currently (in-toto v1.0.0), step declarations have the following
+link:https://github.com/in-toto/specification/blob/v1.0/in-toto-spec.md#431-steps[schema]:
 
 ```
 name: string
 threshold: int
-expectedMaterials: list of artifact rules
-expectedProducts: list of artifact rules
+expected_materials: list of artifact rules
+expected_products: list of artifact rules
 pubkeys: list of authorized key IDs
-expectedCommand: list of strings
+expected_command: list of strings
 ```
 
 This ITE proposes updating this schema to incorporate predicate types.
 
 ```
 name: string
 command: list of strings
-expectedMaterials: list of artifact rules
-expectedProducts: list of artifact rules
-expectedPredicates: list of expectedStepPredicates
+expected_materials: list of artifact rules
+expected_products: list of artifact rules
+expected_predicates: list of expectedStepPredicates
```

Comment:
Note that the new `expectedStepPredicate` schema below proposes `predicateType` as a field -- since this object is new this is not a backcompat break, but stylistically it would be more consistent to declare this as `predicate_type`.

Reply:
I don't have a strong opinion stylistically here as the changes are going to be backwards incompatible anyway. @nmiyake, are you proposing we snake case this for consistency with what we already have in v1 layouts?

At the same time, I've increasingly wondered whether multiple predicates per step is actually an anti-pattern that can be specially handled for something like witness's attestation collection type (cc @jkjell). If we decide to have one predicate type per step, we may actually arrive at a case where ITE-10's changes are backwards compatible, with only the addition of new fields (predicate type), so that may influence this decision.

Reply:
Tracking in https://github.com/in-toto/ITE/issues/58

I think we can merge this in the meantime and decide on the fields + schema / naming separately.